rtl/eclrtl/rtlfield.cpp

```diff
@@ -319,6 +435,11 @@ __int64 RtlSwapIntTypeInfo::getInt(const void * ptr) const
         return rtlReadInt(ptr, length);
 }
 
+double RtlSwapIntTypeInfo::getReal(const void * ptr) const
+{
+    return (double) getInt(ptr);
```

Comment:
Needs to be different depending on whether int is signed or not.

All this code is very similar to the code in deftype/defvalue.  is there any way it could be commoned up?